src/cpp/session/resources/grid/dtviewer.js

```diff
@@ -205,15 +250,28 @@ var renderCellContents = function(data, type, row, meta, clazz) {
 };
 
 var renderCellClass = function (data, type, row, meta, clazz) {
-  var title = null;
-  if (typeof(data) === "string") 
-     title = data.replace(/\"/g, "&quot;");
-  return '<div class="cell">' + 
-         '<div class="' + clazz + '" ' + 
-         (title !== null ? 'title="' + title + '"' : '') +
-         '>' + 
-         renderCellContents(data, type, row, meta, clazz) + '</div>' +
-         '<div class="resizer" data-col="' + meta.col + '" /></div>';
+
+  // render cell contents
+  var contents = renderCellContents(data, type, row, meta, clazz);
+
+  // compute classes for tag
+  var classes = [clazz];
+
+  // treat data with more than 10 characters as 'long'
+  if (contents.length >= 10)
+    classes.push('largeCell');
+
+  // compute title (if any)
+  var title = undefined;
+  if (typeof data === "string")
+    title = data.replace(/"/g, "&quot;");
```

Comment:
Do we need to do this? (it looks like the escaping pass now happens in `createTag`)